test-requirements.txt

```diff
@@ -1,8 +1,8 @@
-sphinx!=1.6.6,!=1.6.7,!=2.1.0,!=3.0.0,!=3.4.2,<2 # BSD
+sphinx!=1.6.6,!=1.6.7,!=2.1.0,!=3.0.0,!=3.4.2,<3 # BSD
```

Comment:
Here it's the official abstract deps proposed by OpenStack and I would suggest no to change it.